app/controllers/choices_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ChoicesController < ApplicationController
+
+  def create
+    @question = Question.find(params[:question])
+    text = params[:text]
+    correct = params[:correct] == '1'
+    Choice.create(:text => text, :correct => correct, :question_id => @question.id)
+    redirect_to question_path(@question)
+  end
+
+  def destroy
+    @question = Question.find(params[:question])
```

Comment:
you can DRY this., my adding custom method and call it before action
EX https://github.cerner.com/HealtheLife/education_library/blob/master/app/controllers/media_controller.rb#L24-L26